deploy/scripts/000100-revoke-roles.ts

```diff
@@ -0,0 +1,81 @@
+import {
+    grantRole,
+    InstanceName,
+    isLive,
+    renounceRole,
+    revokeRole,
+    setDeploymentMetadata
+} from '../../utils/Deploy';
+import { Roles } from '../../utils/Roles';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
+    const { deployer, daoMultisig } = await getNamedAccounts();
+
+    // Grant CarbonController admin roles to dao multisig
+    await grantRole({
+        name: InstanceName.CarbonController,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: daoMultisig,
+        from: deployer
+    });
+
+    await revokeRole({
+        name: InstanceName.CarbonController,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: deployer,
+        from: deployer
+    });
+
+    // Grant Voucher admin roles to dao multisig
+    await grantRole({
+        name: InstanceName.Voucher,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: daoMultisig,
+        from: deployer
+    });
+
+    await revokeRole({
+        name: InstanceName.Voucher,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: deployer,
+        from: deployer
+    });
+
+    // Grant FeeBurner admin roles to dao multisig
+    await grantRole({
+        name: InstanceName.FeeBurner,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: daoMultisig,
+        from: deployer
+    });
+
+    await revokeRole({
+        name: InstanceName.FeeBurner,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: deployer,
+        from: deployer
+    });
+
+    // renounce CarbonController's ROLE_EMERGENCY_STOPPER role from the deployer
+    await renounceRole({
+        name: InstanceName.CarbonController,
+        id: Roles.CarbonController.ROLE_EMERGENCY_STOPPER,
```

Comment:
Perhaps the DAO multisig should be set here explicitly as well?

Reply:
Discussed with Yudi to not set it in the scripts

Reply:
👍